doc/man3/SSL_CTX_set_options.pod

```diff
@@ -200,6 +200,16 @@ this option is set or not CCS messages received from the peer will always be
 ignored in TLSv1.3. This option is set by default. To switch it off use
 SSL_clear_options(). A future version of OpenSSL may not set this by default.
 
+=item SSL_OP_NO_ANTI_REPLAY
+
+By default, when a server is configured for early data (i.e. max_early_data > 0),
+OpenSSL will switch on replay protection. See L<SSL_read_early_data(3)> for a
+description of the replay protection feature. Some applications may be able to
+mitigate the replay risks in other ways and therefore the built in OpenSSL
+functionality is not required. Those applications can turn this feature off by
```

Comment:
I'd rather see something like "and in such cases the built-in OpenSSL functionality is not required", and maybe even a general statement noting that "anti-replay measures are required in order to comply with the TLS 1.3 specification; OpenSSl supplies an implementation that is used by default but allows the application to override the default and provide an application-specific anti-replay mechanism" to try to drive home that this is not optional and should not just be disabled without some replacement mechanism.

Reply:
Ok, I used your text, and added the more general statement too.